### ISSUE TITLE
Null reference when using the ?-> operator with PHP 8

### ISSUE THREAD

**Issue by maicol07** [kalessil/phpinspectionsea#1691]
| Subject        | Details                                                                       |
| :------------- | :---------------------------------------------------------------------------- |
| Plugin         | Php Inspections (EA Extended) 4.0.5 |
| Language level | PHP 8.0                        |

Current behaviour
--
The ?-> raises a Null reference warning with PHP 8

Expected behaviour
--
With PHP 8 the ?-> operator checks that the variable isn't null, so this warning shouldn't appear.

Environment details
---

```
PhpStorm 2021.1 EAP
Build #PS-211.5787.18, built on February 11, 2021
PhpStorm EAP User
Expiration date: March 13, 2021
Runtime version: 11.0.10+9-b1319.6 amd64
VM: Dynamic Code Evolution 64-Bit Server VM by JetBrains s.r.o.
Windows 10 10.0
GC: G1 Young Generation, G1 Old Generation
Memory: 2022M
Cores: 16
Registry: run.processes.with.pty=TRUE, ide.balloon.shadow.size=0
Non-Bundled Plugins: Batch Scripts Support (1.0.12), String Manipulation (8.10.191.000.0), com.chrisrm.idea.MaterialThemeUI (5.7.0), com.github.patou.gitmoji (1.7.0), com.intellij.ideolog (203.0.26.0), com.mallowigi (36.0), com.vexus2.intellij.openingithub (1.0.6), io.github.biezhi.plugins.gitmoji-unicode (1.0.5), com.intellij.plugins.html.instantEditing (211.5787.4), com.github.shiraji.yaemoji (1.0.7), de.espend.idea.laravel (0.15.4), com.kalessil.phpStorm.phpInspectionsEA (4.0.6), de.espend.idea.php.annotation (8.0.0), lv.midiana.misc.phpstorm-plugins.deep-keys (2021.01.05.001), lv.midiana.misc.idea-plugins.deep-js-completion (2020.11.30.001), ru.adelf.idea.dotenv (2021.1), net.king2500.plugins.PhpAdvancedAutoComplete (1.1.0), com.funivan.idea.phpClean (2020.12.22)
```

**Comment by ea-inspections-team** [comment]
Thanks for reporting @maicol07, the issue has been addressed in Php Inspections (EA Extended) v4.0.6.4.
